spec/abci/abci.md

```diff
@@ -238,7 +238,7 @@ via light client.
   - `ChainID (string)`: ID of the blockchain.
   - `ConsensusParams (ConsensusParams)`: Initial consensus-critical parameters.
   - `Validators ([]ValidatorUpdate)`: Initial genesis validators, sorted by voting power.
-  - `AppStateBytes ([]byte)`: Serialized initial application state. Amino-encoded JSON bytes.
+  - `AppStateBytes ([]byte)`: Serialized initial application state. Proto-encoded JSON bytes.
```

Comment:
Not proto encoded, we should add a minimal spec for the supported json. 

We dont assume the json encoding, just `json.rawmessage`